opcua/common/ua_utils.py

```diff
@@ -203,16 +203,8 @@ def data_type_to_variant_type(dtype_node):
     if base.nodeid.Identifier != 29:
         return ua.VariantType(base.nodeid.Identifier)
     else:
-        # we have an enumeration, we need to look at child to find type
-        descs = dtype_node.get_children_descriptions()
-        bnames = [d.BrowseName.Name for d in descs]
-        if "EnumStrings" in bnames:
-            return ua.VariantType.LocalizedText
-        elif "EnumValues" in bnames:
-            return ua.VariantType.ExtensionObject
-        else:
-            raise ua.UaError("Enumeration must have a child node describing its type and values")
-
+        # we have an enumeration, value is a Int32
+        return ua.VariantType.Int32
```

Comment:
from what you wrote today, this is correct. I am curious to see the consequences of that change...